geos/handle.go

```diff
@@ -62,6 +61,7 @@ func newHandle() (*handle, error) {
 		h.release()
 		return nil, errors.New("malloc failed")
 	}
+	C.memset((unsafe.Pointer)(h.errBuf), 0, 1024)
```

Comment:
This was a bug I noticed while I was here. Malloc is not going to actually zero out the memory content... (I've been spoilt by Go).